Assets/Scripts/MeshGenerator.cs

```diff
@@ -10,7 +10,8 @@ public static class MeshGenerator {
         for (int x = 0; x < width; x++) {
             for (int y = 0; y < height; y++) {
                 Vector3 position = new Vector3 (x - width/2 + .5f, heightMap[x, y], y - height/2 + .5f);
-                meshData.AddVertex (position);
+                Vector2 uv = new Vector2 (x/(float)width, y/(float)height);
```

Comment:
Shouldn't the uv's all be straight up (assuming it's flat ish)? Unless im missing something this is "sphere like" uvs?